src/client/Microsoft.Identity.Client/AppConfig/AuthorityInfo.cs

```diff
@@ -141,10 +141,7 @@ internal class AuthorityInfo
         /// <summary>
         /// True if SHA2 and PSS can be used for creating the client credential from a certificate 
         /// </summary>
-        internal bool IsSha2CredentialSupported =>
-            AuthorityType != AuthorityType.Dsts &&
-            AuthorityType != AuthorityType.Generic &&
-            AuthorityType != AuthorityType.Adfs;
+        internal bool IsSha2CredentialSupported => false; 
```

Comment:
I recommend opening a GH issue to track re-enabling this. Not a bad idea to link the issue from the code.

Reply:
https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/4695